src/stories/containers/Entry/AddEntry/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { action } from '@storybook/addon-actions'
+import { Form, message } from 'antd'
+import { has } from 'ramda'
+
+import AddEntryContainer from '../../../../containers/Entry/AddEntry'
+import buildEntry from '../../../../utils/entrySpec'
+import { validateSerialNumberForEntry } from '../../../../utils/validators'
+
+export default {
+  title: 'Containers/Entries',
+  component: AddEntryContainer,
+}
+
+const handleSubmitAction = action('On submit entry form!')
+
+const Template = (args) => {
+  const [alllowInsertSerilNumber, setAlllowInsertSerilNumber] = useState(false)
```

Comment:
```suggestion
  const [alllowInsertSerialNumber, setAlllowInsertSerialNumber] = useState(false)
```

fazer em todas que esteja "SerilNumber"